### PR TITLE
Transcription styles (#1194) and admin fixes (#1193, #1202)

### PR DESCRIPTION
**Pull request by blms**
## In this PR

- Per #1194:
  - Apply Arabic type styles, and conditional type styles using inline lang attributes, to transcriptions
  - Adjust decimal cover
- Per #1193:
  - Apply proper fonts, sizes, line heights, decimal cover to transcription viewer in admin
- Per #1202:
  - Add digital edition switcher to admin (move it to `iiif` bundle from `main`)
- Ensure `alert_controller` excluded from `main` bundle (it should only be in `annotation`; this is why I was getting double `connect`s in Stimulus)

## Questions

- Should this include what we discussed with [`dir` attributes in the HTML](https://www.w3.org/International/questions/qa-html-dir)? If so, should I just apply RTL to all transcriptions for now, or do we want logic to determine it based on language direction?

### CODE REVIEW

File: sitemedia/css/admin-local.css [hunk 1]
```diff
@@ -114,16 +118,45 @@ a#needsreview:target {
...
+    font-family: "FrankRuhl 1924 MF Medium", serif;
+    line-height: 32px;
+}
+/* arabic script */
+.form-row .transcription[lang="ar"] p,
+.form-row .transcription[lang="ar"] li,
+.form-row .transcription[data-lang-script="arabic"] p,
+.form-row .transcription[data-lang-script="arabic"] li {
+    font-family: "Amiri", serif;
+}
```
Comment (rlskoeser) [inline sitemedia/css/admin-local.css:153]: I wish we didn't have to duplicate these, but I guess it isn't that much.

If you think about a good way to consolidate common styles so we can get rid of the duplication at some point, LMK.

File: sitemedia/js/main.js [hunk 2]
```diff
@@ -7,7 +7,7 @@ const application = Application.start();
 const context = require.context(
     "./controllers",
     true,
-    /^.*\/(?!iiif_controller)(?!annotation_controller).*\.js$/
+    /^.*\/(?!iiif_controller)(?!annotation_controller)(?!alert_controller)(?!transcription_controller).*\.js$/
```
Comment (rlskoeser) [inline sitemedia/js/main.js:10]: would you add a comment here about what this is doing? this is what you mean about the double error?

**Review by rlskoeser — APPROVED** [review]
Looks great. I downloaded all the new fonts and tested locally. I checked in search results with records that were tagged as primary language arabic, and I also tested with a transcription that had mixed content, and tagged the arabic section in the transcription editor — it's really obvious when it switches to the arabic font, and it looks great.

One question that I wish had occurred to me sooner: are the Arabic glyphs even in the Hebrew font? Could we just always have the Arabic transcription font applied as a fallback?

To your question about RTL: I think we should put it in the html where appropriate, and I'm pretty sure that all our transcriptions are RTL, so no need to check language.

**Comment by rlskoeser** [comment]
I forgot to test styles in admin. Fonts are being loaded; I think there's a discrepancy in the marker spacing between admin and public site — possibly related to Arabic font size? Two digit line numbers are getting cut off slightly on the right in admin. (I'm testing with PGPID 1242)

**Comment by blms** [comment]
> One question that I wish had occurred to me sooner: are the Arabic glyphs even in the Hebrew font? Could we just always have the Arabic transcription font applied as a fallback?

@rlskoeser That's a really good question. In fact, looking into it, it seems we can just restrict fonts to certain Unicode ranges. Should I just apply both font faces to all transcriptions restricting them by Unicode range? https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face/unicode-range

File: sitemedia/js/main.js (hunk 2 above)
Comment (blms) [reply sitemedia/js/main.js:10]: Yes, the double error was that the stimulus controller for alerts was being loaded twice (once in the `main` bundle and once in the `annotation` bundle).

**Comment by rlskoeser** [comment]
Do the changes in this PR also address https://github.com/Princeton-CDH/geniza/issues/670 ?

(Asking in part because we need to make sure the unicode range includes all of the characters we expect/allow)

I tested that one locally in both Firefox and Vivaldi and I don't see the boxes, although I'm not sure I'm seeing the accents either.

**Comment by blms** [comment]
> Do the changes in this PR also address https://github.com/Princeton-CDH/geniza/issues/670 ?

@rlskoeser Not yet, it's the Goitein and Friedman transcription that has the problems though, I'm still seeing them on Chrome.

I could actually just set that broken character range to a different font like Times New Roman. I'll play around and see what I can do!

**Comment by blms** [comment]
@rlskoeser That solution seems to work really well, including fixing the missing glyphs and the marker spacing. It has the side effect of reducing the repetition as well.

Would you mind taking another look locally to see if it works/if I missed anything? Your checks in the review were really helpful! 

FWIW, I've been testing with PGPIDs 646, 1390, 5468, 6176, 19011, and 36266.

**Review by rlskoeser — APPROVED** [review]
This is great! So glad you found this unicode range font feature, amazing. 

I found a good test case: a letter in JA with Arabic address (PGPID 3271). The Arabic is rendering beautifully without having to tag the language in the editor.

The only thing I can think of we might not be handling is coptic numerals, but I'm not sure we have any transcriptions or characters for those (and not really in scope here, either!). But you have a fallback font configured (and one of the ones Gissoo said is most complete in terms of glyphs), so that seems fine to me.

**Pull request merged**